product_description_without_description_sale/models/product_template.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    use_description_sale = fields.Boolean(
+        "Use Sales Description",
+        help="If selected, Sales Description will be appended"
```

Comment:
```suggestion
        help="If selected, Sales Description will be appended "
```

Reply:
I've added spaces at the end of the sentence.